plugins/woocommerce/readme.txt

```diff
@@ -171,9 +171,9 @@ WooCommerce comes with some sample data you can use to see how products look; im
 
 = 9.3.2 2024-09-18 =
 
-- Fix invalid path error in product importer in Windows (#51456)[https://github.com/woocommerce/woocommerce/pull/51456]
-- Revert - changes related to low stock product notification (#51441)[https://github.com/woocommerce/woocommerce/pull/51441]
-- Fix bug where manually triggering `added_to_cart` event without a button element caused an Exception (#51449)[https://github.com/woocommerce/woocommerce/pull/51449]
+- Fix invalid path error in product importer in Windows [#51456](https://github.com/woocommerce/woocommerce/pull/51456)
+- Revert - changes related to low stock product notification [#51441](https://github.com/woocommerce/woocommerce/pull/51441)
+- Fix bug where manually triggering `added_to_cart` event without a button element caused an Exception [#51449](https://github.com/woocommerce/woocommerce/pull/51449)
```

Comment:
Thanks, and sorry for the earlier miss. On taking a second look, should we format/tweak slightly, so we get something like the following?

```suggestion
- Fix - Improve the product importer's handling of filepaths under Windows [#51456](https://github.com/woocommerce/woocommerce/pull/51456)
- Fix - Revert changes related to low stock product notifications [#51441](https://github.com/woocommerce/woocommerce/pull/51441)
- Fix - Resolve a bug where manually triggering `added_to_cart` event without a button element caused an Exception [#51449](https://github.com/woocommerce/woocommerce/pull/51449)
```

Just thinking that the standard formatting is `Type - Message [PR Link](://)`, and `Revert` as a prefix seems very atypical/not supported by the current changelogger, so the above might be a little more consistent with other changelogs from prior releases.